src/sort.rs

```diff
@@ -68,7 +72,26 @@ pub fn sort_toml(
     let mut first_table = None;
     let mut heading_order: BTreeMap<_, Vec<Heading>> = BTreeMap::new();
     for (idx, (head, item)) in toml.as_table_mut().iter_mut().enumerate() {
-        if !matcher.heading.contains(&head.get()) {
+        let mut table_path: Option<Vec<String>> = None;
```

Comment:
Could this be simplified to just `Vec<String>` with the default being an empty list?

Reply:
refactored.